source/developer/fx-guidelines.rst

```diff
@@ -248,6 +248,16 @@ The color of the on_hover state should be the same as in active state, with opac
 |
 ---------------------------
 
+Display: Tooltips
```

Comment:
Propose moving below `Display: Instructions` to keep the design guidelines section in alphabetical order.

Reply:
Done.